tests/jballerina-unit-test/src/test/resources/test-src/workers/invalid-receive-before-workers.bal

```diff
@@ -2,6 +2,6 @@ function invalidWorkReceiveBeforeWorker() {
   int m1 = <- w1;
   worker w1 {
     int i = 1;
-    i -> default;
+    i -> function;
   }
 }
```

Comment:
Let's add missing new lines of these files with this PR. Please fix in other files as well